test/EndToEndTests/Tests/Client/BindingTests.cs

```diff
@@ -13,27 +13,23 @@ namespace Microsoft.Test.OData.Tests.Client
 
     using Microsoft.Test.OData.Services.TestServices;
     using Microsoft.Test.OData.Services.TestServices.AstoriaDefaultServiceReference;
-#if WIN8 || WINDOWSPHONE
-    using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
-#else
-    using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-#endif
 #if !PORTABLELIB && !SILVERLIGHT

```

Comment:
Possible to remove these portablelib silverlight references while you are doing this?